src/Employer/Employer.Web/ViewModels/Part1/NumberOfPositions/NumberOfPositionsViewModel.cs

```diff
@@ -10,9 +10,12 @@ public class NumberOfPositionsViewModel : VacancyRouteModel
         public PartOnePageInfoViewModel PageInfo { get; set; }
         public ReviewSummaryViewModel Review { get; set; } = new ReviewSummaryViewModel();
         public string NumberOfPositions { get; set; }
+        public bool FromMaSavedFavourites { get; set; }
```

Comment:
The model shouldn't care about MA. Just focus on the minimal amount of stuff you need to render the UI elements.